code/game/objects/items/devices/scanners.dm

```diff
@@ -398,13 +398,14 @@ GENE SCANNER
 			else
 				to_chat(user, "<span class='info'>Blood level: [blood_percent] %, [C.blood_volume] cl, type: [blood_type]</span>")
 
-		var/cyberimp_detect
+		var/list/cyberimp_detect = list()
 		for(var/obj/item/organ/cyberimp/CI in C.internal_organs)
 			if(CI.status == ORGAN_ROBOTIC && !CI.syndicate_implant)
-				cyberimp_detect += "[C.name] is modified with a [CI.name].<br>"
+				cyberimp_detect += CI.name
 		if(cyberimp_detect)
```

Comment:
```suggestion
		if(length(cyberimp_detect))
```

Reply:
Ooo nice, I was looking for the len syntax